pyispyb/core/schemas/laboratories.py

```diff
@@ -0,0 +1,24 @@
+from pydantic_sqlalchemy import sqlalchemy_to_pydantic
+from ispyb import models
+from pyispyb.core.schemas.validators import WordDashSpace
+
+
+PydanticLaboratory = sqlalchemy_to_pydantic(
+    models.Laboratory, exclude={"laboratoryId", "recordTimeStamp"}
+)
+
+
+class LaboratoryBase(PydanticLaboratory):
+    # name, city and country required to be able to check for existing Laboratory in DB (to update or create)
+    # Optionally the system will check also against laboratoryExtPk
+    name: str = WordDashSpace(title="Laboratory Name")
+    address: str
+    city: str
+    country: str
+
+
+class Laboratory(LaboratoryBase):
```

Comment:
Even though we only have the schema for POST here, maybe it would be nice to clearly distinguish schemas meant for POST from schemas meant for GET with some naming convention, as we'll probably eventually have both version with slight differences for every entity (usually ID included for GET but excuded for POST - sometimes some more differences like here). Something like `Laboratory` and `LaboratoryCreate`. What do you think?

Also why is the declaration split in 3 steps?

Reply:
Yes sure, you are right. I will make two separate schema classes Laboratory and LaboratoryCreate. Regarding the 3 steps declaration I think it was not needed.

Reply:
Still undecided about `sqlalchemy_to_pydantic` especially for entity creation. In this case its probably ok, but in other tables we don't just want to blanket make all parameters populate-able. We'll probably want to specify explicit columns and document them

Reply:
I think the idea is to have a `<x>Create` entity, then `<x>` which inherits and enabled `orm_mode = True`. For patch / updates there is `make_optional`, which makes all fields optional in https://github.com/ispyb/py-ispyb/pull/202/files#diff-e9c426778fbe2e8417c3ef97e68c059cca37fb1a8eb356314cef110b3508abf9R21

Reply:
Some duplication here with https://github.com/ispyb/py-ispyb/blob/master/pyispyb/core/schemas/labcontacts.py

Reply:
Removed pydantic from the laboratories.py file and the duplication from labcontacts.py